utils.js

```diff
@@ -165,3 +165,38 @@ class ObjectsMap {
         });
     }
 };
+
+let _currentDesktopsMatches = {};
+// is:
+// @name: desktop string you want to assert if it matches the current desktop env
+//
+// The function examples XDG_CURRENT_DESKTOP and return if the current desktop
+// is part of that desktop string.
+//
+// Return value: if the environment isn't set or doesn't match, return False
+// otherwise, return True.
+//
+// This function is a copy of:
+// https://github.com/endlessm/gnome-shell/blob/master/js/misc/desktop.js
+function is(name) {
```

Comment:
The original commit has the benefit of allowing us to write `Desktop.is()`, but this nuance is lost here. Perhaps it's better to rename this function to `desktopIs()` so that we can call it as `Util.desktopIs('endless')`. What do you think?